discord_bot.py

```diff
@@ -22,10 +21,7 @@ async def on_raw_reaction_add(self, payload):
         description_dict = string_description_to_dictionary(updated_message.embeds[0].description)
         if "Poll" in updated_message.embeds[0].footer.text:
             new_description = get_new_description(description_dict, updated_message.reactions)
-            new_embed = discord.Embed(title=updated_message.embeds[0].title,
-                                      description=new_description,
-                                      colour=discord.Color.orange())
-            new_embed.set_footer(text='Poll'.format(updated_message.id))
+            new_embed = generate_embed(updated_message.embeds[0].title, new_description)
             await updated_message.edit(embed=new_embed)
 
     async def on_raw_reaction_remove(self, payload):
```

Comment:
This function seems to be the same as the on_raw_reaction_add function. Can you abstract this so both call the same function?